extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/config/MicroprofileMetricsConfig.java

```diff
@@ -14,15 +14,19 @@
      * Microprofile Metrics support.
      * <p>
      * Support for Microprofile metrics will be enabled if micrometer
-     * support is enabled, and this value is true. You need to also
-     * include the microprofile api jar to your dependencies:
+     * support is enabled and the MicroProfile Metrics dependency is present:
      *
      * <pre>
      * &lt;dependency&gt;
      *   &lt;groupId&gt;org.eclipse.microprofile.metrics&lt;/groupId&gt;
      *   &lt;artifactId&gt;microprofile-metrics-api&lt;/artifactId&gt;
      * &lt;/dependency&gt;
      * </pre>
+     * <p>
+     * This application uses the MP Metrics API.
```

Comment:
You can probably remove "This application uses the MP Metrics API." from the javadoc.

Reply:
Fixed now